.env

```diff
@@ -64,4 +64,4 @@ ELASTICSEARCH_HOSTS=
 LOG_LEVEL_ROOT=TRACE
 LOG_LEVEL_MONGODB=TRACE
 
-BUILD_CACHE_REPO=openfoodfacts/openfoodfacts-build-cache
+BUILD_CACHE_REPO=openfoodfacts/openfoodfacts-build-cache
```

Comment:
I think this change is unrelated, could you remove it from the PR?

Reply:
this is the last empty line which was removed. It sometimes depends on the IDE.